xds/internal/client/bootstrap/bootstrap.go

```diff
@@ -61,10 +61,10 @@ var gRPCVersion = fmt.Sprintf("%s %s", gRPCUserAgentName, grpc.Version)
 var bootstrapFileReadFunc = ioutil.ReadFile
 
 // Config provides the xDS client with several key bits of information that it
-// requires in its interaction with an xDS server. The Config is initialized
-// from the bootstrap file.
+// requires in its interaction with an management server. The Config is
```

Comment:
Nit: s/an management server/the management server/

Reply:
Done